src/main/java/org/broadinstitute/hellbender/utils/fragments/FragmentCollection.java

```diff
@@ -0,0 +1,144 @@
+package org.broadinstitute.hellbender.utils.fragments;
+
+import org.broadinstitute.hellbender.utils.pileup.PileupElement;
+import org.broadinstitute.hellbender.utils.pileup.ReadPileup;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.util.*;
+import java.util.function.Function;
+
+/**
+ * Represents the results of the reads -> fragment calculation.
```

Comment:
This class needs much better top-level docs -- this does not adequately explain the purpose of the class.